### DISCUSSION TITLE
Welcome to go-dvote Discussions!

### DISCUSSION THREAD

**Discussion by p4u** [vocdoni/vocdoni-node#61] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ferranrego** [comment]
Great 🚀 !

I'm Ferran, Communication Lead @ Vocdoni.

I'm more active in other repositories like Docs, Vochain explorer, vocdoni-manager-frontend or mobile, but happy to be around, learn and maybe catch -and report- bugs on the fly!
